improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -721,10 +666,11 @@ def test_use_mask_cube_occurrences_not_masked(self):
                 [1.0, 1.0, 1.0, 1.0, 1.0],
             ]
         )
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2),), num_grid_points=5, num_time_points=1
-        )
-        cube = iris.util.squeeze(cube)
+
+        data = np.ones((5, 5), dtype=np.float32)
+        data[2, 2] = 0
+        cube = set_up_variable_cube(data, spatial_grid="equalarea",)
+
         mask_cube = cube.copy()
         mask_cube.data = np.array(
```

Comment:
Nice-to-have - you can set this up more concisely as:
```
mask_cube = cube.copy(data=np.ones((5, 5), dtype=np.float32))
```

Reply:
Done.